aspnet/mvc/controllers/testing/sample/TestingControllersSample/src/TestingControllersSample/ClientModels/IdeaDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace TestingControllersSample.ClientModels
+{
+    public class IdeaDTO
```

Comment:
True, but not intent of this topic. Could be worth a note and link to https://docs.asp.net/en/latest/mvc/models/formatting.html in the article, though.